app/views/catalog/collection_search.html.erb

```diff
@@ -10,7 +10,7 @@
       View current collections below. Click an image or link for a full description.
     </div>
 <%
-	@current_unit =  @current_unit.gsub!("UCSD","") if @current_unit && @current_unit.instance_of?(String) && @current_unit.include?("UCSD")
+	@current_unit =  @current_unit.gsub!("UCSD ","") if @current_unit && @current_unit.instance_of?(String) && @current_unit.include?("UCSD")
```

Comment:
why is this included in this PR? The primary change looks 👍 to me, though I've never seen that child selector syntax before. But, it looks like it works!